modelling/src/main/java/org/axonframework/modelling/command/inspection/AnnotatedAggregate.java

```diff
@@ -410,8 +410,6 @@ public Object handle(Message<?> message) throws Exception {
     private Object handle(CommandMessage<?> commandMessage) throws Exception {
         List<AnnotatedCommandHandlerInterceptor<? super T>> interceptors =
                 inspector.commandHandlerInterceptors((Class<? extends T>) aggregateRoot.getClass())
-                         .filter(chi -> chi.canHandle(commandMessage))
```

Comment:
Yes, sorting is not this class' responsibility anymore.
The implementation has been changed to chain the invocations itself. That means the "chain" will check if the interceptor at a specific can perform the invocation. If not, it will proceed without invoking the interceptor.

Reply:
Aaah of course, makes total sense.